ansible/roles/stack-sunbird/templates/sunbird_learner-service.env

```diff
@@ -109,3 +109,5 @@ sunbird_subdomain_keycloak_base_url={{sunbird_subdomain_keycloak_base_url | d(''
 sunbird_cert_service_base_url=http://cert-service:9000
 quartz_shadow_user_migration_timer={{sunbird_quartz_shadow_user_migration_timer}}
 sunbird_otp_allowed_attempt={{sunbird_otp_allowed_attempt|default(2)}}
+SUNBIRD_KAFKA_URL={{kafka_urls}}
```

Comment:
where are you referencing the topic name?

Reply:
https://github.com/project-sunbird/sunbird-lms-service/blob/release-3.0.0/service/conf/logback.xml

Reply:
it is being referenced for a variablized env name: https://github.com/project-sunbird/sunbird-lms-service/blob/def5b4b78d34200e394cf43382a0dda291781372/service/conf/logback.xml#L33

Reply:
Looks good